exporter/loadbalancingexporter/loadbalancer.go

```diff
@@ -198,3 +208,27 @@ func (lb *loadBalancerImp) Exporter(endpoint string) (component.Component, error
 
 	return exp, nil
 }
+
+func checkMultipleResolversProvided(cfg component.Config) (bool, error) {
```

Comment:
I'm not entirely comfortable with this change in this PR. I think it has to be on its own PR.

Reply:
Sure, I'll raise a new PR once this is merge with the increment.